### ISSUE TITLE
Rename "Efficient High-Volume Streaming" doc to "Producers and Consumers: Efficient High-Volume Streaming"

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#1931]
|<img alt="spiv's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| spiv reported|
|-|-|
|Trac ID|trac#1931|
|Type|enhancement|
|Created|2006-07-17 04:55:12Z|
|Branch|https://github.com/twisted/twisted/tree/rename-doc-titles-1931|

The reason to do this is so that people who have heard about producers and consumers can find them easily in the Developer Guide table of contents.  Just "Efficient High-Volume Streaming" is overly abstract.  "Cred: Pluggable Authentication" is an example of a title that already gets this right.

Possibly we should review all documentation titles for this, e.g. "Tips for writing tests for Twisted code" ought to mention Trial in the title.

<details><summary>Searchable metadata</summary>

```
trac-id__1931 1931
type__enhancement enhancement
reporter__spiv spiv
priority__normal normal
milestone__ 
branch__branches_rename_doc_titles_1931 branches/rename-doc-titles-1931
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1153112112000000 1153112112000000
changetime__1219511880000000 1219511880000000
version__None None
owner__ 
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set owner to edsuom|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra commented|
|-|-|

(In [24559]) Branching to 'rename-doc-titles-1931'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra removed owner<br>@thijstriemstra set status to `new`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun set owner to @thijstriemstra|
|-|-|

Looks good.  If http://buildbot.twistedmatrix.com/builders/documentation/builds/766 succeeds, please merge.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_6"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [24631]) Merge rename-doc-titles-1931: reviewed titles for core documentation and made them less abstract.

Author: thijs
Reviewer: exarkun
Fixes: [#1931](https://github.com/twisted/twisted/issues/1931)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
